### PR TITLE
Add ZOPE_FORM_MEMORY_LIMIT environment variable to configure

### PR DESCRIPTION
**Pull request by ericof**
Ref: https://github.com/plone/Products.CMFPlone/issues/3848

### CODE REVIEW

**Review by fredvd — APPROVED** [review]
Lgtm

File: skeleton/docker-entrypoint.sh [hunk 1]
```diff
@@ -27,6 +27,8 @@ fi
 [ -z ${DEFAULT_ZPUBLISHER_ENCODING+x} ] && export DEFAULT_ZPUBLISHER_ENCODING=utf-8
 [ -z ${COMPILE_MO_FILES+x} ] && export COMPILE_MO_FILES=true
 [ -z ${DEBUG_MODE+x} ] && export DEBUG_MODE=off
+[ -z ${ZOPE_DOS_PROTECTION+x} ] && export ZOPE_DOS_PROTECTION=4MB
```
Comment (davisagli) [inline skeleton/docker-entrypoint.sh]: I would call this ZOPE_FORM_MEMORY_LIMIT. It is not the only setting related to DOS protection that Zope now supports.

**Review by davisagli — CHANGES_REQUESTED** [review]

**Review by mauritsvanrees — APPROVED** [review]
LGTM, thanks.
David's suggestion makes sense.

File: skeleton/docker-entrypoint.sh (hunk 1 above)
Comment (ericof) [reply skeleton/docker-entrypoint.sh]: Good call, I will rename it

Comment (ericof) [reply skeleton/docker-entrypoint.sh]: @davisagli Solved

**Review by davisagli — APPROVED** [review]

**Pull request merged**